javascript/exercises/cashRegister.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Javascript: The Cash Register</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link href="../../common/css/bootstrap.css" rel="stylesheet">
+  </head>
+ <body>
+<div class="container">
+
+<h2 class="page-header">The Cash Register</h2>
+
+<p><em>  </em></p>
+<ul>
+ <li>Write a function called <code>cashRegister</code> that takes an object containing shopping cart values for various customers. The function should return an array that contains the name of the person and the sum of their cart. (ie [ [ 'Ernie', 237.54 ], [ 'Bert', 3.38 ] ])''
```

Comment:
Can we change the assignment to ask to output an array instead of changing the solution?